src/streamlink/plugins/chzzk.py

```diff
@@ -0,0 +1,281 @@
+            if playlist.is_iframe:
+                continue
+
+            if playlist.stream_info.resolution and playlist.stream_info.resolution.height:
+                stream_name = f"{playlist.stream_info.resolution.height}p"
+
+            if not stream_name:
+                continue
+            uris = [playlist.uri]
+            for media in playlist.media:
+                if media.uri:
+                    uris.append(media.uri)
+
+            substreams = [ChzzkHLSStream(session, uri, channel) for uri in uris]
+            streams[stream_name] = MuxedStream(session, *substreams, format="mpegts", copyts=True)
+
+        return streams
+
+
+class ChzzkAPI:
+    _CHANNELS_LIVE_DETAIL_URL = "https://api.chzzk.naver.com/service/v2/channels/{channel_id}/live-detail"
+    _VIDEOS_URL = "https://api.chzzk.naver.com/service/v2/videos/{video_id}"
+
+    def __init__(self, session):
+        self._session = session
+
+    def _query_api(self, url, *schemas):
+        return self._session.http.get(
+            url,
+            acceptable_status=(200, 404),
+            schema=validate.Schema(
+                validate.parse_json(),
+                validate.any(
+                    validate.all(
+                        {
+                            "code": int,
+                            "message": str,
+                        },
+                        validate.transform(lambda data: ("error", data["message"])),
+                    ),
+                    validate.all(
+                        {
+                            "code": 200,
+                            "content": dict,
+                        },
+                        validate.get("content"),
+                        *schemas,
+                        validate.transform(lambda data: ("success", data)),
+                    ),
+                ),
+            ),
+        )
+
+    def get_live_detail(self, channel_id):
+        return self._query_api(
+            self._CHANNELS_LIVE_DETAIL_URL.format(channel_id=channel_id),
+            {
+                "status": str,
+                "liveId": int,
+                "liveTitle": validate.any(str, None),
+                "liveCategory": validate.any(str, None),
+                "channel": validate.all(
+                    {"channelName": str},
+                    validate.get("channelName"),
+                ),
+                "livePlaybackJson": validate.all(
+                    str,
+                    validate.parse_json(),
+                    {
+                        "media": [
+                            {
+                                "mediaId": str,
+                                "protocol": str,
+                                "path": validate.url(),
+                            },
+                        ],
+                    },
+                    validate.get("media"),
+                    validate.transform(lambda media: [(m["mediaId"], m["protocol"], m["path"]) for m in media]),
```

Comment:
How can I use validate.union_get to extract the values of properties from the array inside `media`? I'm looking for a way to efficiently retrieve these values in a tuple.